ansible/group_vars/tag_Environment_beta

```diff
@@ -2,26 +2,30 @@ register_domain: register.gov.uk
 register_settings:
   country:
     custodian_name: Tony Worron
-    history_page_url: https://registers-history.herokuapp.com/country
+    history_page_url: https://register-status.herokuapp.com/registers/country
     similar_registers:
       - territory
   datatype:
     custodian_name: Paul Downey
+    history_page_url: https://register-status.herokuapp.com/registers/datatype
   field:
     custodian_name: Paul Downey
+    history_page_url: https://register-status.herokuapp.com/registers/field
   local-authority-eng:
     custodian_name: Stephen McAllister
-    history_page_url: https://registers-history.herokuapp.com/local-authority-eng
+    history_page_url: https://register-status.herokuapp.com/registers/local-authority-england
```

Comment:
Are we okay with naming not being consistent between the register and the history page?